pages/common/git-brv.md

```diff
@@ -0,0 +1,9 @@
+# git brv
+
+> Print a list of branches, sorted by their date since last commit.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-brv>.
+
+- Print a list of branches showing date, latest commit and latest commit hash for each branch:
```

Comment:
"latest" could probably be omitted, wdyt?
```suggestion
- Print a list of branches showing date, latest commit and commit hash for each branch:
```

Reply:
Yes, I will have to do it in a different commit since this is out of date

Reply:
Actually, I have added @navarroaxel edit